scpd/lms/static/css/vendor/su-identity.css

```diff
@@ -0,0 +1,302 @@
+  clear: both;
+}
+
+#su-header .row:after,
+#su-footer .row:after {
+  clear: both;
+}
+
+/* Background gradient */
+#su-header {
+  background: #8C1515;
+  box-shadow: inset 0 -4px 8px -2px rgba(0, 0, 0, 0.2); /* Horizontal offset , vertical offset, blur, internal spacing */
+  min-height: 30px;
+}
+
+#su-footer,
+#su-footer .su-container {
+  background: #8C1515;
+  color: #e3dfd5;
+  box-shadow: inset 0 4px 8px -2px rgba(0, 0, 0, 0.2); /* Horizontal offset , vertical offset, blur, internal spacing */
+}
+
+/* Prevent flush left */
+#su-header .su-container img {
+  padding: 4px 0 0 10px;
+}
+
+#su-footer a {
+  color: #e3dfd5;
+}
+
+#su-footer a:hover {
+  text-decoration: underline;
+}
+
+/* Prevent horizontal scroll bar */
+#su-footer .su-container {
+  padding: 3px 7px 20px;
+}
+
+#su-footer .su-container .su-row {
+  margin: 30px 0 0;
+}
+
+/* Align logo and text */
+#su-footer-logo {
+  margin: 0 0 10px;
+  padding: 0;
+}
+
+#su-footer-text {
+  margin-top: 0;
+}
+
+#su-footer-text ul {
+  margin: 2px 0 0;
+  padding: 0;
+  line-height: 1em;
+}
+
+#su-footer-text ul li {
+  display: block;
+  list-style: none;
+}
+
+#su-footer-text ul li a {
+  font: 13px 'Source Sans Pro','Helvetica Neue',Helvetica,Arial,sans-serif;
+  text-decoration: none;
+}
+
+#su-footer p.copyright {
+  font: 11px 'Source Sans Pro','Helvetica Neue',Helvetica,Arial,sans-serif;
+  margin: 0;
+  padding: 30px 0 0;
+  line-height: 1.5em;
+  color: #e3dfd5;
+}
+
+#su-footer p.copyright span {
+  color: #e3dfd5;
+}
+
+/* Footer two columns */
+#su-footer .span2 {
+  width: 48%;
+  float: left;
+}
+
+#su-footer .span10 {
+  width: 48%;
+  float: right;
+}
+
+#su-footer img {
+  margin: 0 auto;
+  display: block;
+}
+
+/* Clear the floats */
+#su-footer .clear {
+  clear: both;
+}
+
+/*
+========================================================
+Media queries
+*/
+
+@media screen and (min-width: 480px) {
+}
+
+@media screen and (min-width: 768px) {
+  /* Prevent double background gradient */
+  #su-footer .su-container {
+    background:none;
+    box-shadow:none;
+  }
+
+  #bottom-text {
+    margin-top:2px;
+  }
+
+  /* Remove left padding */
+  #su-header .su-container img {
+    padding:4px 0 0;
+  }
+
+  /* Bootstrap style */
+  #su-header .su-container,
+  #su-footer .su-container {
+    width:724px;
+  }
+
+  #su-footer .su-container .su-row {
+    margin:30px 0 0 -20px;
+  }
+  /* Footer single column */
+  #su-footer .span2 {
+    float:left;
+    width:104px;
+    padding-left:8px; /* Prevent flush left */
+  }
+  /* Bootstrap style */
+  #su-footer .span10 {
+    width:600px;
+    margin-left:20px;
+    margin:0 12px 0 20px;
+    float:left;
+  }
+
+  #su-footer img {
+    margin:0 auto;
+  }
+
+  #su-footer ul li.alt {
+  	clear:right;
+  }
+  /* Footer two column */
+  #su-footer ul {
+    width:100%;
+    float:left;
+  }
+
+  #su-footer ul li {
+  	width:auto;
+  	margin:0 13px 3px 0;
+  	padding:0;
+  	float:none;
+  	clear:left;
+    display:inline-block;
```

Comment:
Formatting/styling is wonky through this file.
I suspect there's a mix of tabs and spaces, 2 space vs 4 space indents,
many rules don't have a space after the colon, etc.

Let's normalize this.

Related, for files like this that shouldn't vary from instance to instance, we should be symlinking instead of creating multiple copies.